Sources/PointFree/TODO.swift

```diff
@@ -429,10 +419,11 @@ extension IO {
 
 import Dispatch
 
-func zip<A>(_ parallels: [Parallel<A>]) -> Parallel<[A]> {
+func sequence<A>(_ parallels: [Parallel<A>]) -> Parallel<[A]> {
```

Comment:
I defined a bunch of sequences of flipping array with `Parallel`, `IO`, and `EitherIO`